src/api-utils.js

```diff
@@ -2,14 +2,24 @@ import axios from "axios";
 
 const API = axios.create({ baseURL: "https://liams-nc-news.onrender.com/api" });
 
-export const fetchArticles = () => {
-  return fetch("https://liams-nc-news.onrender.com/api/articles")
-    .then((res) => {
-      return res.json();
-    })
-    .then((res) => {
-      return res.articles;
-    });
+export const fetchArticles = (currentTopic) => {
+  if (currentTopic === "All") {
+    return fetch("https://liams-nc-news.onrender.com/api/articles")
+      .then((res) => {
+        return res.json();
+      })
+      .then((res) => {
+        return res.articles;
+      });
+  } else {
+    return fetch(`https://liams-nc-news.onrender.com/api/articles?topic=${currentTopic}`)
```

Comment:
You could just change the string that you're going to use in fetch instead of repeating the whole promise chain here.